Sources/PubNubChat/Helpers/CustomJSONData.swift

```diff
@@ -37,7 +37,9 @@ public struct VoidCustomData: Codable, Hashable,
   UserCustomData,
   ChannelCustomData,
   MemberCustomData,
-  MessageCustomData
+  MessageCustomData,
+  MessageCustomDataContentKey,
+  MessageCustomDataCustomKey
```

Comment:
Can this be possibly be left as `MessageCustomData`, and then the new value can be `MessageContentData`?

Reply:
`MessageContentData` is very good! 👍  

Btw, do you have an idea how to solve it better? As you can see, [MessageCustomData](https://github.com/pubnub/chat-components-ios/blob/features/chat-message/Sources/PubNubChat/Helpers/CustomJSONData.swift#L55-L58) became a separate protocol that contains two generic types for `content` and `custom`. Is it OK for you? The reason I did this was to prevent introducing a new generic type in `ChatCustomData` so there are no breaking changes and it still looks in [the same way](https://github.com/pubnub/chat-components-ios/blob/features/chat-message/Sources/PubNubChat/Helpers/CustomJSONData.swift#L64) like before. So applying your idea with `MessageContentData`, I thought about something like this:

```
public protocol ??? {
  associatedtype Content: MessageContentData = VoidCustomData
  associatedtype Custom: MessageCustomData = VoidCustomData
}
```
```
public protocol ChatCustomData {
  associatedtype User: UserCustomData = VoidCustomData
  associatedtype Channel: ChannelCustomData = VoidCustomData
  associatedtype Member: MemberCustomData = VoidCustomData
  associatedtype Message: ??? = VoidCustomData
}
```
What would be the best name for `???`

Reply:
I do not like this but probably just MessageData, but at this point it is super confusing because name MessageCustomData and MessageData does not tell whats the difference. Hard to tell

Reply:
Ok, I changed it [this](https://github.com/pubnub/chat-components-ios/blob/features/chat-message/Sources/PubNubChat/Helpers/CustomJSONData.swift#L55-L58) way so now `MessageCustomData` has two associated types that refers to payload.
I hope `MessagePayloadContentData` and `MessagePayloadCustomData` tell you that:

```
public protocol MessageCustomData {
  associatedtype Content: MessagePayloadContentData = VoidCustomData
  associatedtype Custom: MessagePayloadCustomData = VoidCustomData
}
```